enterprise/frontend/test/metabase-enterprise/formatting/whitelabel.cy.spec.js

```diff
@@ -218,44 +229,44 @@ describeWithToken("formatting > whitelabel", () => {
       });
     });
 
-    it("should reflect logo change on admin's dashboard", () => {
-      signInAsAdmin();
+    it("changes should reflect on admin's dashboard", () => {
       cy.visit("/");
       checkLogo();
     });
 
-    it("should reflect logo change while signed out", () => {
+    it("changes should reflect while signed out", () => {
+      signOut();
       cy.visit("/");
       checkLogo();
     });
 
-    it("should reflect logo change on user's dashboard", () => {
+    it("changes should reflect on user's dashboard", () => {
       signInAsNormalUser();
       cy.visit("/");
       checkLogo();
     });
   });
 
-  it("should add a custom favicon", () => {
-    signInAsAdmin();
-    cy.visit("/admin/settings/whitelabel");
-
-    cy.server();
-
-    cy.findByPlaceholderText("frontend_client/favicon.ico").type(
-      "https://cdn.ecosia.org/assets/images/ico/favicon.ico",
-    );
-    cy.get("ul")
-      .eq(2)
-      .click("right");
-    cy.findByText("Saved");
-    checkFavicon();
+  describe("favicon", () => {
```

Comment:
This test actually doesn't really work - well, it works, but the favicon isn't applied to browsers EE#439

Reply:
Hm, it seems that it does - at least in Chrome that I'm using for Cypress. The second (middle) tab is `localhost:4000` - a home page opened with the favicon applied. 

![image](https://user-images.githubusercontent.com/31325167/104322355-28948c80-54e5-11eb-898b-d9d39e2aa422.png)

Reply:
Seems to be working now on Chromium - didn't work some months ago - and it doesn't work on Firefox.

Reply:
:) :(